google/cloud/forseti/scanner/audit/location_rules_engine.py

```diff
@@ -184,13 +184,18 @@ def _build_rule(cls, rule_def, rule_index):
                 raise errors.InvalidRulesSchemaError(
                     'Missing field "{}" in rule {}'.format(field, rule_index))
 
-        applies_to = rule_def.get('applies_to')
-        for resource_type in applies_to:
+        applies_to = {}
+
+        for applies_dict in rule_def.get('applies_to'):
+            resource_type = applies_dict['type']
+
             if resource_type not in SUPPORTED_LOCATION_RESOURCE_TYPES:
                 raise errors.InvalidRulesSchemaError(
                     'Unsupported applies to type "{}" in rule {}'.format(
                         resource_type, rule_index))
 
+            applies_to[resource_type] = applies_dict['resource_ids']
```

Comment:
Can we make this backward compatible? e.g. if 'resource_ids' is not defined then we can use '*' as the default value.

Reply:
Done.